smarty/templates/main.tpl

```diff
@@ -159,11 +159,12 @@
                                 <img width=17 src="{$baseurl}/images/help.gif">
                             </a>
                         </li>
-                        <li>
-                            <p class="navbar-text">
-                                &nbsp;&nbsp;  Sites: {$user.Sites} &nbsp;
-                            </p>
+                        <li class="nav">
+                            <a href="#" data-toggle="tooltip" title="{$user.SitesTooltip}" style="padding-right:25px;">
```

Comment:
@driusan 

I got this from line 169 (a few lines lower). Is it problematic there as well?